### PR TITLE
caps, grammar, etc

### PR DESCRIPTION
**Pull request by bbrendon**
Lots of tiny changes. Enjoy!! :)
There is more to do but I had to stop. Seemed like it was taking longer than expected.

### CODE REVIEW

File: docs/guide_gettingstarted.md [hunk 1]
```diff
@@ -23,13 +23,17 @@ Install the server, [choose the best path](install_considerations.md)
 * [ ] Add users to Permission Groups `Settings menu > User Administration`
 * [ ] Decide on "Disable Auto Login for Remote Control and Remote background" `Settings menu > Global Settings > Meshcentral`
 
-## Every 75 days
+### Maintenance
```
Comment (silversword411) [inline docs/guide_gettingstarted.md:26]: These menu levels are there for the TOC to be arranged by:

Time Range
-> Categories in time range

2023-05-07_172723 - brave_Getting_Started_Guide_-_Tactical_RMM_Documentation

I don't think making it a descending tree is the right layout.

Everything else looks good!

**Review by silversword411 — CHANGES_REQUESTED** [review]
Tweak that, everything else looks good, thanks! :)

File: docs/guide_gettingstarted.md (hunk 1 above)
Comment (bbrendon) [reply docs/guide_gettingstarted.md]: I reverted the page

**Review by silversword411 — COMMENTED** [review]
Fixed outside review

**Pull request merged**